internal/archive/fsops_test.go

```diff
@@ -293,9 +331,15 @@ var _ = Describe("FSOPS", func() {
 		)
 	})
 
-	It("getRelativePath", func() {
-		Ω(getRelativePath(getFullPath("abc", "xyz", "fff"),
-			filepath.Join(getFullPath()))).Should(Equal(string(filepath.Separator) + filepath.Join("abc", "xyz", "fff")))
+	var _ = Describe("getRelativePath", func() {
```

Comment:
Shouldn't this return the full path when it isn't relative to the prefix?

Reply:
added test